third_party/istio-1.4-latest

```diff
@@ -1 +1 @@
-istio-1.4.4
\ No newline at end of file
+istio-1.4.6
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
istio-1.4.6

```

Reply:
It is a symlink so we do not need to add trailing newline.